hpobench/dependencies/ml/ml_benchmark_template.py

```diff
@@ -0,0 +1,290 @@
+import time
```

Comment:
my main question is about the location of this file
especially if we are creating a subfolder `ml` under `dependencies`, might as well have it at the benchmark folder?